src/System.Net.Http/tests/FunctionalTests/TestHelper.cs

```diff
@@ -93,7 +93,7 @@ public static byte[] ComputeMD5Hash(byte[] data)
 
         public static Task WhenAllCompletedOrAnyFailed(params Task[] tasks)
         {
-            return TaskTimeoutExtensions.WhenAllOrAnyFailed(tasks);
+            return TaskTimeoutExtensions.WhenAllOrAnyFailed(tasks, PlatformDetection.IsArmProcess || PlatformDetection.IsArm64Process ? 30_000 : 10_000);
```

Comment:
Should this just use PassingTestTimeoutMilliseconds?

Reply:
updated. The value is slightly different but works fine in my test runs.